src/orca/services/nextflowtower/models/enums.py

```diff
@@ -0,0 +1,15 @@
+class TaskStatus:
```

Comment:
Nit: Should something like this be used? https://docs.python.org/3/library/enum.html

Reply:
added in latest commit